sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/DoNotCatchNullReferenceException.cs

```diff
@@ -0,0 +1,20 @@
+using System;
```

Comment:
I was actually wondering for those cases and I thought that in this case the user will mark the issue as `won't fix` because they know it is fine in this case. WDYT?